src/derive.rs

```diff
@@ -33,66 +35,143 @@ declare_lint! {
     "deriving `Hash` but implementing `PartialEq` explicitly"
 }
 
+/// **What it does:** This lint warns about explicit `Clone` implementation for `Copy` types.
+///
+/// **Why is this bad?** To avoid surprising behaviour, these traits must agree and the behaviour
```

Comment:
> s/override/overridden

Rhhaa, that’s only the sixth choice for *overrided* in Vim’s spell checker :sweat_smile: